tekton/cronjobs/robocat-certificates-on-demand/cronjob.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: folder-cd-trigger
+spec:
+  suspend: true
+  schedule: "1 1 1 1 1"
```

Comment:
once a year?

Reply:
Heh, no, it's `suspend: true`, so it only runs on demand.
It must be run once after the cert-manager is deployed.
The second one in the table here: https://github.com/tektoncd/plumbing/blob/2c53795759c4dc5d2d928279da20cc002dc32fb0/robocat/README.md#run-the-cronjobs